server/user.go

```diff
@@ -38,13 +38,40 @@ func getOrCreateUser(ctx context.Context, sub string, info *oidc.UserInfo) (*Use
 	return &user, nil
 }
 
-type GetMeOutput struct {
+type GetUserInput struct {
+	ID string `path:"id"`
+}
+
+type GetUserOutput struct {
 	Body User
 }
 
-func GetMe(ctx context.Context, input *struct{}) (*GetMeOutput, error) {
-	user := getCurrentUser(ctx)
-	return &GetMeOutput{Body: *user}, nil
+func GetUser(ctx context.Context, input *GetUserInput) (*GetUserOutput, error) {
+	return getUser(ctx, &input.ID)
+}
+
+func GetMe(ctx context.Context, input *struct{}) (*GetUserOutput, error) {
+	return getUser(ctx, nil)
+}
+
+func getUser(ctx context.Context, sub *string) (*GetUserOutput, error) {
+	out := &GetUserOutput{}
+	if sub != nil {
+		user, err := getOrCreateUser(ctx, *sub, nil)
```

Comment:
get or create?

Reply:
It no longer upserts the user when info is nil